app/controllers/api/v1/book_suggestions_controller.rb

```diff
@@ -0,0 +1,20 @@
+module Api
+  module V1
+    class BookSuggestionsController < ApplicationController
+      def create
+        params[:book_suggestion][:user_id] = current_user.id if user_signed_in?
```

Comment:
Remove this line. For this endpoint you need to be authenticated, add a `before_action`

Reply:
According with the Trello's Card its not necessary to be authenticated. He says "Wolox Training QUIERE que se implemente el CREATE de un BookSuggestion sin autenticación para que un usuario invitado en la plataforma pueda crear una sugerencia." So, I made that for, if the user is authenticated save his id in the BookSuggest. In the card before it, he request me that BookSuggestion can save an User Reference. This way I'm accomplish with both cards are requesting me.

Reply:
Done, I changed it. Now, only authenticated users can send a Book Suggestion.